validation/provisioning/rke2/provisioning_node_driver_test.go

```diff
@@ -167,8 +168,15 @@ func (r *RKE2NodeDriverProvisioningTestSuite) TestProvisioningRKE2ClusterDynamic
 				clusterObject, err := provisioning.CreateProvisioningCluster(tt.client, provider, credentialSpec, clusterConfig, machineConfigSpec, nil)
 				require.NoError(r.T(), err)
 
+				logrus.Info("ClusterID:", clusterObject.ID)
```

Comment:
Why is this being added?

Reply:
sorry this is for a separate PR, can be ignored for now.

Reply:
yup, that's the plan.